probe/process/walker_linux.go

```diff
@@ -83,6 +88,11 @@ func (w *walker) Walk(f func(Process, Process)) error {
 			continue
 		}
 
+		fileDescriptors, err := readFileDescriptors(path.Join(w.procRoot, filename, "fd"))
```

Comment:
I would inline the function call. It's just two lines and the parameter is not obvious.